### DISCUSSION TITLE
Cheat menu improvements

### DISCUSSION THREAD

**Discussion by axx0** [axx0/Civ2-clone#40] [Ideas]
I've been thinking of ways of improving existing and adding new cheat options & features.

For starters I could replace existing listbox in "Edit technologies" with tickboxes. That means I would have to use a custom dialog, but it would make the whole thing much easier to use.

**Comment by reubene** [comment]
That's a good idea, keen to see how you implement it. 

Are there other places we could use a dialog with tick boxes? I think there is already one for customize rules in the new game sequence?

**Comment by axx0** [comment]
Yeah there are some in Edit King etc.
The idea is to simplify things. E.g. changing money now requires 1) choosing a civ, 2) text input. There is no reason for this not to be in a single window with 7 lines of text input for each civ.

JCivED for Civ1 is one example how to squeeze as much options as possible into a single window.
![image](https://user-images.githubusercontent.com/21365802/156888651-6a73fb52-0246-42c7-a5b3-6fc16c1c6d62.png)

**Reply by reubene** [reply]
Yeah that makes sense, being able to switch back and forth between civs when editing is useful. A dialog that presents more info like jciv Ed does is a better idea.

The way the original works with multiple dialogs is not very friendly for scenario editing

**Comment by Blakeley00** [comment]
Good idea!! It's quite frustrating having to go through all those windows when trying to setup technologies and king info for all 7 civs!
